frontend/packages/dev-console/src/components/import/advanced/ResourceLimitSection.tsx

```diff
@@ -0,0 +1,75 @@
+import * as React from 'react';
+import FormSection from '../section/FormSection';
+import { ResourceLimitField } from '../../formik-fields';
+
+const ResourceLimitSection: React.FC = () => {
+  const cpuUnits: { [key: string]: string } = {
```

Comment:
Can we create enums for these and move it to types instead?

Reply:
Yes, we can create an ```enum ```for these.